docs/guides/latest/hooks/configuring-hooks.md

```diff
@@ -0,0 +1,216 @@
+| `host`        | `Optional` | Specify the name of the host where the GET request will be sent.  If you don't specify this field, Stash will use the hook executor Pod IP as host.                                                                                                                                      |
+| `port`        | `Required` | Specify the port where the HTTP server is listening. You can specify a numeric port or the name of the port in case of the HTTP server is running in the same pod where the hook will be executed. If you specify the port name, you must specify the `containerName` field of the hook. |
+| `path`        | `Required` | Specify the path to access on the HTTP server.                                                                                                                                                                                                                                           |
+| `scheme`      | `Required` | Specify the scheme to use to connect with the host.                                                                                                                                                                                                                                      |
+| `httpHeaders` | `Optional` | Specify a list of custom headers to set in the request.                                                                                                                                                                                                                                  |
+
+**Examples:**
+
+- Hook with `host` and `port` are specified:
+
+```yaml
+preBackup:
+  httpGet:
+    host: my-host.demo.svc
+    port: 8080
+    path: "/"
+    scheme: HTTP
+```
+
+- Hook to extract `host` and `port` from the hook executor pod:
+
+```yaml
+preBackup:
+  httpGet:
+    port: my-port
+    path: "/"
+    scheme: HTTP
+  containerName: my-container
+```
+
+- Hook for sending custom header in the HTTP request:
+
+```yaml
+preBackup:
+  httpGet:
+    host: my-host.demo.svc
+    port: 8080
+    path: "/"
+    scheme: HTTP
+    httpHeaders:
+    - name: "User-Agent"
+      value: "stash/0.9.x"
+```
+
+## HTTPPost
+
+`httpPost` hook allows sending an HTTP POST request to an HTTP server before and after the backup/restore process. The following configurable fields are available for `httpPost` hook.
+
+| Field         | Type       | Usage                                                                                                                                                                                                                                                                             |
```

Comment:
We don't mention the data types in these fields. I think that might be a problem.

Reply:
I wanted to mention data type. But, we can't do that when there is nested type.

Reply:
done